node-test/src/test/scala/org/bitcoins/node/NeutrinoNodeWithWalletTest.scala

```diff
@@ -0,0 +1,111 @@
+package org.bitcoins.node
+
+import akka.actor.Cancellable
+import org.bitcoins.core.crypto.DoubleSha256DigestBE
+import org.bitcoins.core.currency._
+import org.bitcoins.node.networking.peer.DataMessageHandler
+import org.bitcoins.rpc.client.common.BitcoindVersion
+import org.bitcoins.server.BitcoinSAppConfig
+import org.bitcoins.testkit.BitcoinSTestAppConfig
+import org.bitcoins.testkit.fixtures.UsesExperimentalBitcoind
+import org.bitcoins.testkit.node.NodeUnitTest.NeutrinoNodeFundedWalletBitcoind
+import org.bitcoins.testkit.node.{NodeTestUtil, NodeUnitTest}
+import org.bitcoins.wallet.api.UnlockedWalletApi
+import org.scalatest.exceptions.TestFailedException
+import org.scalatest.{DoNotDiscover, FutureOutcome}
+
+import scala.concurrent.duration._
+import scala.concurrent.{Future, Promise}
+
+@DoNotDiscover
```

Comment:
ping? Should this be here?

Reply:
It should we still don't have a statically linked Neutrino-enabled bitcoind binary for CI, so this test will break in CI.